src/Build/Evaluation/IntrinsicFunctions.cs

```diff
@@ -61,14 +53,6 @@ internal static double Subtract(double a, double b)
             return a - b;
```

Comment:
Aside, feels like these guys would be neater in expression body form. It's not my codebase though

Reply:
This PR intentionally has a very narrow focus but in a different PR it may make sense to change to expression body form.